src/main/java/com/sem/btrouble/model/Model.java

```diff
@@ -123,6 +125,51 @@ public static void restartRoom() {
             p.moveTo(getCurrentRoom().getSpawnPositionX(), getCurrentRoom().getSpawnPositionY());
         }
     }
+    
+    public static ArrayList<PowerUp> getPowerUps() {
+        return powers;
+    }
+    
+    public static void addPowerUp(PowerUp power) {
+        powers.add(power);
+    }
+    
+    public static void clearPowerUps() {
+        powers.clear();
+    }
+    
+    public static ArrayList<PowerUp> getShortPower() {
+    	return powersshort;
+    }
+    
+    public static void addShortPowerUp(PowerUp power) {
+    	powersshort.add(power);
+    }
+    
+    public static void deleteShortPower(PowerUp power) {
+    	ArrayList<PowerUp> powers = new ArrayList<PowerUp>();
+    	for(int i = 0; i < powersshort.size(); i++) {
+    		PowerUp listPower = powersshort.get(i);
+    		if(power instanceof LifePowerUp) {
```

Comment:
if(power instanceof...) zou je in de klasse zelf af kunnen handelen dan handelt Java zelf af welke actie hij voor welke klasse hij moet doen.